packages/hooks/use-throttle-render/index.ts

```diff
@@ -1,31 +1,53 @@
 import { onMounted, ref, watch } from 'vue'
+import { isNumber, isObject } from '@element-plus/utils'
 
+import type { SkeletonThrottle } from '@element-plus/components'
 import type { Ref } from 'vue'
 
-export const useThrottleRender = (loading: Ref<boolean>, throttle = 0) => {
+export const useThrottleRender = (
+  loading: Ref<boolean>,
+  throttle: SkeletonThrottle = 0
+) => {
   if (throttle === 0) return loading
-  const throttled = ref(false)
+  const initVal = isObject(throttle) ? throttle.initVal ?? false : false
```

Comment:
It seems that there is no need for false to fall back here.
```suggestion
  const initVal = isObject(throttle) ? throttle.initVal : false
```

Reply:
If remove `throttle.initVal ?? false` , `useThrottleRender` will return `ref<boolean | undefined>`, But I think `useThrottleRender` of return `ref<boolean>` express more clear meaning.

![image](https://github.com/user-attachments/assets/d79f7af9-24e0-4123-92e9-5e05e027b9bc)

![image](https://github.com/user-attachments/assets/8d2e0fc2-c3e3-42c1-a63d-09d25f328d1b)

Reply:
I have thought about modifying it to this `const initVal = isObject(throttle) && Boolean(throttle.initVal)`